### PR TITLE
IS-753: include suspended status in active preps

### PR DESCRIPTION
**Pull request by leeheonseung**

### CODE REVIEW

File: iconservice/prep/data/prep_container.py [hunk 1]
```diff
@@ -53,12 +54,12 @@ def _set_dirty(self, on: bool):
     def is_flag_on(self, flags: 'PRepFlag') -> bool:
         return (self._flags & flags) == flags
 
-    def size(self, active_prep_only: bool = False) -> int:
+    def size(self, active_suspended_prep_only: bool = False) -> int:
```
Comment (soobokjin) [inline iconservice/prep/data/prep_container.py]: I think 'active_prep_only' is a proper name. In prep container, 'active' refers both ACTIVE and SUSPENDED.

Comment (soobokjin) [reply iconservice/prep/data/prep_container.py]: I understood why you have defined like this.

File: iconservice/prep/data/prep_container.py [hunk 2]
```diff
@@ -188,12 +191,17 @@ def get_by_address(self, address: 'Address') -> Optional['PRep']:
...
         return self._prep_dict.get(address)
 
-    def get_preps(self, start_index: int, size: int) -> List['PRep']:
+    def get_preps(self, start_index: int, size: int, active_only: bool = False) -> List['PRep']:
         """Returns active P-Reps ranging from start_index to start_index + size - 1
 
         :return: P-Rep list
         """
-        return self._active_prep_list[start_index:start_index + size]
+        if active_only:
```
Comment (soobokjin) [inline iconservice/prep/data/prep_container.py]: logic is ok, but the meaning is opposite

File: iconservice/prep/data/prep_container.py [hunk 3]
```diff
@@ -151,18 +154,18 @@ def replace(self, new_prep: 'PRep') -> Optional['PRep']:
 
         return old_prep
 
-    def contains(self, address: 'Address', active_prep_only: bool = True) -> bool:
+    def contains(self, address: 'Address', active_suspended_prep_only: bool = True) -> bool:
```
Comment (soobokjin) [inline iconservice/prep/data/prep_container.py]: To retain consistency, integrate 'active_suspended_prep_only' default value to False.

Comment (leeheonseung) [reply iconservice/prep/data/prep_container.py]: spec is changed, that container defined ACTIVE = ACTIVE | SUSPENDED.

File: iconservice/prep/data/prep_container.py (hunk 2 above)
Comment (leeheonseung) [reply iconservice/prep/data/prep_container.py]: fix logic

**Pull request closed without merging**